app/code/Magento/Integration/Model/ResourceModel/Oauth/Token/RequestLog.php

```diff
@@ -46,27 +46,41 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     protected function _construct()
     {
         $this->_init('oauth_token_request_log', 'entity_id');
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
+     * @param string $userName
+     * @param int $userType
      */
     public function getFailuresCount($userName, $userType)
     {
+        $date = (new \DateTime())->setTimestamp($this->dateTime->gmtTimestamp());
+        $dateTime = $date->format(\Magento\Framework\Stdlib\DateTime::DATETIME_PHP_FORMAT);
```

Comment:
Why you didn't use `\Magento\Framework\Stdlib\DateTime\DateTime::gmtDate`?

Reply:
Since previously in the code of incrementing the count, you can see in the incrementFailuresCount and clearExpiredFailures function, the same has been used. So, I think if it would cause any time differences in the timezone, so better stick to the one which has been used previously.